elftools/elf/dynamic.py

```diff
@@ -73,13 +73,14 @@ def __str__(self):
 class Dynamic(object):
     """ Shared functionality between dynamic sections and segments.
     """
-    def __init__(self, stream, elffile, stringtable, position):
+    def __init__(self, stream, elffile, stringtable, position, empty):
```

Comment:
Could you please add a comment explaining what empty is/means as a parameter here?

While you're at it, might I ask you to add a brief description of the ctor's parameters in its docstring :-) ?